test/index.test.ts

```diff
@@ -6,11 +6,13 @@
  */
 import {
   BaseClient,
+  CacheManagerKeyv,
   CacheManagerRedis,
   ClientConfig,
   COMMERCE_SDK_LOGGER_KEY,
   commonParameterPositions,
   getObjectFromResponse,
+  ICacheManager,
```

Comment:
What is the use of ICacheManager import? If not used, remove it to avoid unused imports warnings